OTAnalytics/plugin_prototypes/eventlist_exporter/eventlist_exporter.py

```diff
@@ -0,0 +1,168 @@
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Iterable
+
+import pandas as pd
+
+from OTAnalytics.application.datastore import EventListExporter
+from OTAnalytics.domain.event import (
+    DIRECTION_VECTOR,
+    EVENT_COORDINATE,
+    OCCURRENCE,
+    SECTION_ID,
+    Event,
+)
+from OTAnalytics.domain.section import Section
+
+OTC_EXCEL_FORMAT_NAME = "Excel (OpenTrafficCam)"
+OTC_CSV_FORMAT_NAME = "CSV (OpenTrafficCam)"
+
+
+class EventListDataFrameBuilder:
+    def __init__(self, events: Iterable[Event], sections: Iterable[Section]):
+        self._events = events
+        self._sections = sections
+        self._df = self._convert_to_dataframe(events)
+
+    def _convert_to_dataframe(self, events: Iterable[Event]) -> pd.DataFrame:
+        return pd.DataFrame([event.to_dict() for event in events])
+
+    def build(self) -> pd.DataFrame:
+        self._convert_occurence_to_seconds_since_epoch()
+        self._split_columns_with_lists()
+        self._add_section_names()
+        return self._df
+
+    def _convert_occurence_to_seconds_since_epoch(self) -> None:
+        epoch = pd.Timestamp("1970-01-01")
+        occurence = pd.to_datetime(self._df[OCCURRENCE])
+        self._df[f"{OCCURRENCE}_sec"] = (occurence - epoch).dt.total_seconds()
```

Comment:
The occurrence of an event should already have a `...timestamp...` method to convert it to a timestamp. We should use the same conversion at all places.

Reply:
This would mean unpacking the (large) eventlist dataframe, calling the timestamp function on the occurence and creating the dataframe again. Or using an apply method on the dataframe. This would be much slower than using the builtin pd.Timestamp(). This is a prototype - until the core of OTAnalytics is faster, I would like to leave it as it is, but add a comment here. Do you agree?

Reply:
Or do we already create a separate dataframe for the eventlist internally?

Reply:
Not now, it will be done after profiling.

Reply:
> This would mean iterating over the (large) eventlist. Or using an apply method on the dataframe. This would be much slower than using the builtin pd.Timestamp(). This is a prototype - until the core of OTAnalytics is faster, I would like to leave it as it is, but add a comment here. Do you agree?

True, I mixed it with the detections where the timestamp is a property of the objects and is also available in the dataframe as column.